contentcuration/contentcuration/tasks.py

```diff
@@ -35,8 +40,51 @@
 # runs the management command 'exportchannel' async through celery
 
 
+# Tasks with 'test' in their name are written specifically to test the Async Task API
+@app.task(bind=True, name='test_task')
+def test_task(self):
```

Comment:
Haha I like the return value! These seem to be polluting the production tasks namespace module. I'm trying to think of a way though, on how to move test tasks to a different module. I guess my question is: how can we isolate the tasks into their own namespace?

Reply:
I should be able to add another task file and add it into the list of locations to search.

Reply:
What I ended up doing was moving these files into another module, that gets imported only when `settings.RUNNING_TESTS` is set. This way they don't appear in the main module and don't get added to the task list on production.